packages/inspection-capture-web/src/PhotoCapture/PhotoCapture.tsx

```diff
@@ -123,6 +126,7 @@ export function PhotoCapture({
   ...initialCameraConfig
 }: PhotoCaptureProps) {
   useI18nSync(lang);
+  const { authToken, vehicleType, steeringWheel } = useMonkAppState();
```

Comment:
I don't think we should use `useMonkAppState` in the `PhotoCapture` component :
- If the `vehicleType`, `steeringWheel` etc are defined in the `useMonkAppState`, it means that they already are set in the monitoring and analytics (thanks to `useAppStateMonitoring` and `useAppStateAnalytics`).
- I asked you to also set these value in the PhotoCapture component in the case that the user is specifically NOT using the `useMonkAppState` hook, but still using the `PhotoCapture` component.
- Of course the `PhotoCapture` component does not know the vehicle type and steering wheel position and it's fine, but at least it knows the inspection ID and the auth token.